src/routes/orders.js

```diff
@@ -1,95 +1,90 @@
 const express = require("express");
+const bodyParser = require("body-parser");
+let jsonParser = bodyParser.json();
 let router = express.Router();
 
-const fs = require("fs");
-const path = require("path");
-const shortid = require("shortid");
-
-let productsDataBase = require("../db/all-products.json");
-let usersDataBase = require("../db/all-users.json");
-
-const getProducts = productsOrder => {
-  const productsOrderIds = productsOrder.products;
-  const getProductsResult = [];
-
-  productsOrderIds.map(item => {
-    const getOneProduct = productsDataBase.find(
-      element => element.id === Number(item)
-    );
-    if (getOneProduct) {
-      getProductsResult.push(getOneProduct);
-    }
-  });
-
-  return getProductsResult;
+const mongoose = require("mongoose");
+const Order = require("../../schemas/orders");
+
+const checkOrder = orderToCheck => {
```

Comment:
Try to use some validation library instead of manual implementation. Even now it's hard to understand what you trying to validate